mmengine/visualization/vis_backend.py

```diff
@@ -610,3 +611,176 @@ def close(self):
         """close an opened tensorboard object."""
         if hasattr(self, '_tensorboard'):
             self._tensorboard.close()
+
+
+@VISBACKENDS.register_module()
+class MLFlowVisBackend(BaseVisBackend):
```

Comment:
Suggest renaming `MLFlowVisBackend` to `MLflowVisBackend` because the official name is `MLflow`.

Reply:
reflected.